exasol_integration_test_docker_environment/lib/test_environment/create_ssl_certificates_task.py

```diff
@@ -0,0 +1,124 @@
+    volume_name = luigi.Parameter()
+
+    def run_task(self):
+        self.volume_info = None
+        if self.reuse:
+            self.logger.info("Try to reuse volume %s", self.volume_name)
+            try:
+                self.volume_info = self.get_volume_info(reused=True)
+            except Exception as e:
+                self.logger.warning(f"Tried to reuse volume {self.volume_name}, but got Exeception {e}. "
+                                    "Fallback to create new volume.")
+        if self.volume_info is None:
+            self.volume_info = self.create_docker_volume()
+            self.create_certificate()
+
+        self.return_object(self.volume_info)
+
+    def register_required(self):
+        self.test_container_image_future = \
+            self.register_dependency(self.create_child_task(task_class=DockerTestContainerBuild))
+
+
+    def get_volume_info(self, reused: bool) -> DockerVolumeInfo:
+        with self._get_docker_client() as docker_client:
+            volume_properties = docker_client.api.inspect_volume(self.volume_name)
+            if volume_properties['Name'] == self.volume_name:
+                return DockerVolumeInfo(volume_name=str(self.volume_name),
+                                        mount_point=volume_properties['Mountpoint'], reused=reused)
+
+    def create_docker_volume(self) -> DockerVolumeInfo:
+        self.remove_volume(self.volume_name)
+        with self._get_docker_client() as docker_client:
+            volume = docker_client.volumes.create(
+                name=self.volume_name,
+            )
+            volume_info = self.get_volume_info(reused=False)
+        return volume_info
+
+    def remove_volume(self, volume_name):
+        try:
+            with self._get_docker_client() as docker_client:
+                docker_client.volumes.get(volume_name).remove()
+                self.logger.info("Removed volume %s", volume_name)
+        except docker.errors.NotFound:
+            pass
+
+    def cleanup_task(self, success):
+        if (success and not self.no_cleanup_after_success) or \
+                (not success and not self.no_cleanup_after_failure):
+            try:
+                self.logger.info(f"Cleaning up volume %s:", self.volume_name)
+                self.remove_volume(self.volume_name)
+            except Exception as e:
+                self.logger.error(f"Error during removing volume %s: %s:", self.volume_name, e)
+
+    def create_certificate(self) -> None:
+        template_str = pkg_resources.resource_string(
+            "exasol_integration_test_docker_environment",
+            "docker_db_config/create_certificate.sh")  # type: bytes
+        template = Template(template_str.decode("utf-8"))
+        rendered_template = template.render(#HOST_NAME='db_container_exasol_test.db_network_exasol_test',
+                                            HOST_NAME=f"{self.db_container_name}.{self.network_name}",
```

Comment:
Extract into variable or function to make it more obvious that you construct the host name that way